src/test/java/core/basesyntax/dao/impl/FruitDaoImplTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.dao.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FruitDaoImplTest {
+    private FruitDao fruitDao;
+
+    @BeforeEach
+    void setUp() {
+        fruitDao = new FruitDaoImpl();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.fruitStorage.clear();
+    }
+
+    @Test
+    void getStorage_AfterAddingFruitsReturnsCorrectData_Ok() {
+        fruitDao.putToStorage("apple", 20);
+        fruitDao.putToStorage("banana", 30);
+        fruitDao.putToStorage("lemon", 2);
+        Map<String, Integer> expected = new HashMap<>();
+        expected.put("apple", 20);
+        expected.put("banana", 30);
+        expected.put("lemon", 2);
```

Comment:
If you use strings many times better save it to constants

Reply:
done